io/net/htx_nic_devices.py.data/htx_nic_devices.yaml

```diff
@@ -0,0 +1,9 @@
+mdt_file: 'net.mdt'
+host_ip: x.xx.xxx.xx
+peer_ip: x.xx.xxx.xy
+peer_user: "root"
+peer_password: "******"
```

Comment:
Agreed, but since we run tests one after the other, and other are in a specific format, it would be good to follow the same format :)